comms/tio/Portfile

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           meson 1.0
+
+name                tio
+version             1.45
+revision            1
+categories          comms
+maintainers         nomaintainer
+platforms           darwin
+license             GPL-2
+
+homepage            https://tio.github.io
+master_sites        https://github.com/tio/tio/releases/download/v${version}
+use_xz              yes
+
+description         Simple TTY terminal I/O application
+long_description    tio is a simple serial device tool which \
+                    features a straightforward command-line \
+                    and configuration file interface to easily \
+                    connect to serial TTY devices for basic \
+                    I/O operations.
+
+checksums           sha256  4e72a946792346f86504865ed2923688f4dd9281905bb81e76ba3fd6f3b96014 \
```

Comment:
You also need to add the `rmd160` checksum.  You can use `port bump tio` to automatically fill in the correct checksum, or you can calculate it manually using `openssl`: `openssl dgst -rmd160 <file>`